app/controllers/users/phone_setup_controller.rb

```diff
@@ -45,6 +48,10 @@ def set_setup_presenter
       )
     end
 
+    def setup_voice_preference?
+      params[:otp_delivery_preference].to_s.strip == 'voice'
```

Comment:
What are we expecting to have to `.strip` from the parameter?

Reply:
The style of `.to_s.strip` on params is seen at a few locations in the codebase. It's not strictly necessary, perhaps in some cases it removes leading or trailing whitespaces. I was sticking to house style / historical precedent.

Reply:
The other examples I see of `.to_s.strip` are for field values which would be provided by a user, like an OTP code, which seems reasonable to strip whitespace that a user may enter. Since we're expecting this to be passed through the URL and not by user input, I don't know that we need to worry as much about normalizing the value.

Reply:
Removed.